pandas/io/formats/csvs.py

```diff
@@ -188,6 +198,39 @@ def save(self):
                 for _fh in handles:
                     _fh.close()
 
+    def _index_label_encoder(self):
+        """Encode index label if it is not False.
+
+        Returns:
```

Comment:
Can you conform docstring to the pandas standard?

Reply:
thanks, changed! and i also add several more tests! @WillAyd